### PR TITLE
Rename package to {cfr}

### LINKED ISSUE #47: Rename package to {cfr}
This issue is to suggest a change of name for the package to match the functionality included in it. The name {cfr} is available, and this package could add more CFR estimation methods such as that in https://pubmed.ncbi.nlm.nih.gov/16076827/

### LINKED ISSUE #53: Add package logo
This issue it to request that the package logo be added as an SVG file. The logo and name are still liable to change based on PR #47.

### PR DESCRIPTION
**Pull request by pratikunterwegs**
This PR renames the package to {cfr} from {datadelay}, and fixes #47.

This PR also:
1. Adds a package logo with the new name, and fixes #53,
2. Updates the pkg maintainer to @pratikunterwegs, and adds @TimTaylor, @joshwlambert, and @Bisaloo as reviewers - could you please say whether you're okay with this.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -8,3 +8,4 @@ inst/doc
 /doc/
 /Meta/
 scratch.R
+pkgdown/
```
Comment (Bisaloo) [inline .gitignore:11]: This folder can contain important files we want to commit. Do you have any issues that lead you to add it here?

File: R/prepare_data.R [hunk 2]
```diff
@@ -48,7 +48,7 @@ prepare_data <- function(data, ...) {
 #'
 #' @export
 #' @return A data.frame suitable for disease severity estimation functions
-#' provided in `{datadelay}`, with the columns "date", "cases", and "deaths".
+#' provided in `{cfr}`, with the columns "date", "cases", and "deaths".
```
Comment (Bisaloo) [inline R/prepare_data.R:51]:
```suggestion
#' provided in \pkg{cfr}, with the columns "date", "cases", and "deaths".
```

**Review by Bisaloo — APPROVED** [review]
Could you also please submit PRs to:
- the r-universe repo to update the pkg name and link
- the hex-stickers repo with the logo in png & svg

File: .gitignore (hunk 1 above)
Comment (pratikunterwegs) [reply .gitignore:11]: Currently it only contains the directoy `favicon/` with icons of different sizes - should these be committed? I can allow tracking if so.

Comment (pratikunterwegs) [reply .gitignore:11]: Converting this to an issue for now - can be added to tracking if it becomes necessary, but I don't see a `pkgdown` directory in our other repos so keeping it out for now.

**Pull request merged**